app/v2/models/response/retrieveBalanceAndTransactions/FinanceDetails.scala

```diff
@@ -19,21 +19,16 @@ package v2.models.response.retrieveBalanceAndTransactions
 import api.models.domain.TaxYear
 import play.api.libs.functional.syntax.toFunctionalBuilderOps
 import play.api.libs.json.{JsPath, Json, Reads, Writes}
-import v2.models.response.retrieveBalanceAndTransactions.financeDetailTypes.mapping
 
 case class FinanceDetails(taxYear: String,
-                          documentId: String,
-                          chargeType: Option[String],
-                          mainType: Option[String],
+                          chargeDetail: Option[ChargeDetail],
                           taxPeriodFrom: Option[String],
                           taxPeriodTo: Option[String],
                           contractAccountCategory: Option[String],
```

Comment:
"contractAccountCategory" is no longer included in specs

Reply:
removed